scripts/build/libc/picolibc.sh

```diff
@@ -0,0 +1,116 @@
+# This file adds functions to build the Picolibc library
+# Copyright © 2020 Keith Packard
+# Licensed under the GPL v2 or later. See COPYING in the root of this package
+#
+# Edited by Keith Packard <keithp@keithp.com>
+#
+
+picolibc_start_files()
+{
+    CT_DoStep INFO "Installing C library headers & start files"
+    CT_DoExecLog ALL cp -a "${CT_SRC_DIR}/picolibc/newlib/libc/include/." \
+    "${CT_HEADERS_DIR}"
+    if [ "${CT_ARCH_XTENSA}" = "y" ]; then
+        CT_DoLog EXTRA "Installing Xtensa headers"
+        CT_DoExecLog ALL cp -r "${CT_SRC_DIR}/picolibc/newlib/libc/sys/xtensa/include/."   \
+                               "${CT_HEADERS_DIR}"
+    fi
```

Comment:
Is this valid for picolibc? Taking a quick look at the sources I don't see a `newlib/libc/sys/xtensa` path.

Reply:
Ah, I didn't really review what that line did -- it's from the newlib script. Not useful for picolibc.